content/events/2025-in-person-conference/Code_of_Conduct.md

```diff
@@ -0,0 +1,127 @@
+events, inappropriate physical contact, and unwelcome sexual attention.
+
+Participants asked to stop any harassing behaviour are expected to comply
+immediately.
+
+Be careful in the words that you choose. Remember that words can be offensive
+to those around you. Offensive jokes are not acceptable in the Nordic-RSE.
+Excessive swearing is not appropriate.
+
+
+### Enforcement
+
+If a participant at a Nordic-RSE event violates this code of conduct,
+organisers may take any action they deem appropriate, including a
+warning the offender, removing them from the workshop with no refund, or 
+banning the offender from future Nordic-RSE events.
+
+If a member of Nordic RSE engages in behaviour that violates this code of conduct,
+the board of the association at the recommendation of the Code of Conduct committee
+may take any action they deem appropriate, including warning the offender or revoking
+their membership without refund of the membership fee.
+
+
+### Reporting
+
+If someone makes you or anyone else feel unsafe or unwelcome, please report it
+as soon as possible. Harassment and other code of conduct violations reduce the
+value of our event for everyone. We want you to be happy at our event. People
+like you make our event a better place.
+
+You can make a report either personally or anonymously.
+
+
+#### Anonymous Report
+You can make [this online form](https://nordicrse.wufoo.com/forms/z5zh7d61domtah/).
+
+We can't follow up an anonymous report with you directly, but we will fully investigate
+it and take whatever action is necessary to prevent a recurrence.
+
+
+#### Personal Report
+
+You can make a personal report by:
+
+ * Contacting any member of the Code of Conduct committee
+    * NN (email)
+    * NN (email)
+    * NN (email)
+ * Contacting any conference organizer
+ * Contacting any member of the [Nordic-RSE board](/about/governance/)
+
+When taking a personal report, organizers will ensure you are safe and cannot be
+overheard. They may involve other event staff to ensure your report is managed properly.
+Once safe, we'll ask you to tell us about what happened. This can be upsetting, but
+we'll handle it as respectfully as possible, and you can bring someone to support you.
+You won't be asked to confront anyone and we won't tell anyone who you are.
+
+Our team will be happy to help you contact hotel/venue security, local law enforcement,
+local support services, provide escorts, or otherwise assist you to feel safe for the
+duration of the event. We value your attendance.
+
+
+### Contact Information
+
+**In an emergency call XXX**. This includes medical, police and rescue services.
+
+Organizers: secretary@nordic-rse.org
+Venue help line: XXX XXXX XXX
+
+Emergency (Medical, police or anything else urgent): XXX
+Non-emergency medical care: XXXXX
+Local law enforcement (Non Urgent): XXX XXXX XXX
+Crime victim support: XXX XXX
+Local sexual assault hot line: XXX XXXX XXX, website
+
+Local taxi company: XXXX XXXX XXXX
```

Comment:
People can google this...

Reply:
It's in case you need to get away quickly. I can google it, though.